V2_MIGRATION_GUIDE.md

```diff
@@ -60,164 +63,129 @@ Additionally, any classes that implemented `ParameterizableRequest` or `AuthRequ
 - The `com.auth0.android.result.Delegation` class has been removed. This was used as the result of the request to the [/delegation](https://auth0.com/docs/api/authentication#delegation) Authentication API legacy endpoint, disabled as of June 2017.
 - The `com.auth0.android.authentication.request.DelegationRequest` class has been removed. This was used to represent the request to the legacy Authentication API [/delegation](https://auth0.com/docs/api/authentication#delegation) endpoint, disabled as of June 2017.
 - The `com.auth0.android.util.Telemetry` class has been renamed to `com.auth0.android.util.Auth0UserAgent`.
-- The `com.auth0.android.request.AuthorizableRequest` class has been removed. You can achieve the same result using the method in Request: `Request#addHeader("Authorization", "Bearer TOKEN_VALUE")`.
+- The `com.auth0.android.request.AuthorizableRequest` class has been removed. You can achieve the same result using the `addHeader("Authorization", "Bearer TOKEN_VALUE")`.
```

Comment:
nit: If we don't use "the ... method" we should remove "the".
```suggestion
- The `com.auth0.android.request.AuthorizableRequest` class has been removed. You can achieve the same result using `addHeader("Authorization", "Bearer TOKEN_VALUE")`.
```

Also, should we use `Bearer {TOKEN_VALUE}` to be consistent with how you are passing these in the rest of the file?

Reply:
Updated to remove superfluous "the" and update example as suggested for consistency.